messageroute.go

```diff
@@ -51,52 +63,77 @@ type (
 	// message handler and a specific message type.
 	MessageRoute interface{ isMessageRoute() }
 
-	// HandlesCommandRoute describes a route for a handler that handles a
-	// [Command] of a specific type.
+	// HandlesCommandRoute is a [HandlerRoute] that represents a handler's
+	// ability to handle [Command] messages of a specific type.
+	//
+	// Avoid constructing values of this type directly; use [HandlesCommand]
+	// instead.
 	HandlesCommandRoute struct{ Type reflect.Type }
 
-	// ExecutesCommandRoute describes a route for a handler that executes a
-	// [Command] of a specific type.
+	// ExecutesCommandRoute is a [HandlerRoute] that represents a handler's
+	// ability to execute [Command] messages of a specific type.
+	//
+	// Avoid constructing values of this type directly; use [ExecutesCommand]
+	// instead.
 	ExecutesCommandRoute struct{ Type reflect.Type }
 
-	// HandlesEventRoute describes a route for a handler that handles an
-	// [Event] of a specific type.
+	// HandlesEventRoute is a [HandlerRoute] that represents a handler's
+	// ability to handle [Event] messages of a specific type.
+	//
+	// Avoid constructing values of this type directly; use [HandlesEvent]
+	// instead.
 	HandlesEventRoute struct{ Type reflect.Type }
 
-	// RecordsEventRoute describes a route for a handler that records an
-	// [Event] of a specific type.
+	// RecordsEventRoute is a [HandlerRoute] that represents a handler's
+	// ability to record [Event] messages of a specific type.
+	//
+	// Avoid constructing values of this type directly; use [RecordsEvent]
+	// instead.
 	RecordsEventRoute struct{ Type reflect.Type }
 
-	// SchedulesTimeoutRoute describes a route for a handler that schedules a
-	// [Timeout] of a specific type.
+	// SchedulesTimeoutRoute is a [HandlerRoute] that represents a handler's
+	// ability to schedule [Timeout] messages of a specific type.
+	//
+	// Avoid constructing values of this type directly; use [SchedulesTimeout]
+	// instead.
 	SchedulesTimeoutRoute struct{ Type reflect.Type }
 )
 
 type (
-	// HandlesCommandOption is an option that affects the behavior of the route
-	// returned by [HandlesCommand].
+	// HandlesCommandOption is an option that modifies the behavior of
+	// [HandlesCommand].
+	//
+	// This type exists for forward-compatibility.
 	HandlesCommandOption struct{}
 
-	// ExecutesCommandOption is an option that affects the behavior of the route
-	// returned by [ExecutesCommand].
+	// ExecutesCommandOption is an option that modifies the behavior of
+	// [HandlesEvent].
+	//
+	// This type exists for forward-compatibility.
 	ExecutesCommandOption struct{}
 
-	// HandlesEventOption is an option that affects the behavior of the route
-	// returned by [HandlesEvent].
+	// HandlesEventOption is an option that modifies the behavior of
+	// [RecordsEvent].
```

Comment:
Is this one correct also?
```suggestion
	// HandlesEventOption is an option that modifies the behavior of
	// [HandlesEvent].
```